### ISSUE TITLE
OP Build6: Gunslinger

### ISSUE THREAD

**Issue by Wildlovelies** [trowl223/Compound_X#71]
-gunslinger machine gun pistol build with +40 dmg to pistols...

Gunslinger is the only class that can get the full +40 dmg to pistols, as far as ammo counts go, since they only use secondaries, primary _mags_ hold more than just one _mag_ per slot, extended mags are not that expensive.  shooting 3-4 shots per half action per gun at 79+ dmg per hit...

**Comment by trowl223** [comment]
> Additionally the same can be said for every class that uses SMGs (Only Soldiers, Doctors, Priests, Pilots, Adepts and Paladins), +3 rounds per half action of firing doubles their max rate of fire, doubling damage. The +30% miss chance is countered by more availability on an SMG to reduce that miss chance with upgrades.


From Issue #58. Additionally, please mention the fact that you are using feats to reach the +40 damage. Gunslingers only have a +10 damage native to the class.

**Comment by bleehu** [comment]
The more detailed these bug reports are, the more precisely we can balance the fixes. =)

**Comment by bleehu** [comment]
Attn. @Wildlovelies: 
What feats is this using?
What level is this at?
What attachments is this using?
How much range does this character have?
How much health and armor does this character have?
What race is this character?

**Comment by trowl223** [comment]
@Wildlovelies

**Comment by Wildlovelies** [comment]
LVL4 Cyborg
Str:6 +2 from cyborg
Per:6 +1 from cyborg                 
fort:5 
Char:9                                 
Int:4
Dex:9
Luck:4
Class Passives: Pistoleer lvl1, + 10 dmg, +10m, +10% miss (malice)
Feats: LVL2: Dual wield, combat expertise guns, LVL4: Pistoleer 2&3


Total money, 1700 +100*luck - 800, +1000 per lvl (3) = 4300
Po8 phantomsx2 = 1140
pistol slingx2 = 200
heavy barrelx2= 200
red laserx2= 200
1.2 pistol sightsx2= 250
Lightened machine slidex2: 1600
total so far = 4190, so I have 710 left
Ceramic Plate, Steel backed $220
30%
15 + d10 DR
4 primary + 4 secondary mags 
Leaves 490 for gask mask and extended mags for all slots

PO8 stats (remember dual wield)
40m + 10m + 5m =  55m (base, class passive, attachment)
DMG: 47 + 10 + 30 = 87(base, class passive, feats. )
miss%: 35 + 10 + 30 - 15 - 10 - 10 - 10 - 4 =  26 (base, class passive (malice), dual wield, expertise guns, attachments, perception modifier) 

Firing dex/3 shots per half action per gun: up to 12 each gun if aimed (12)(87)(2) = 2088 dmg per round 
(12 shots per gun at 87 dmg a piece and two guns doing so) 


.
.
.

Lvl 6 feat would be akimbo,
lvl 6 miss chance: 
miss%: 35 + 10 + 10 - 15 - 10 - 10 - 10 - 4 =  6 *add 20% if targeting two targets = 26%* (base, class passive (malice), akimbo, expertise guns, attachments )

lvl 8 feat would be hands of an artist for better hit chance.
lvl 8 would also take:
Acquired: Additional Plating
Prerequisites: Marksman, Gunslinger
Enables the use of Medium Armour by the listed classes.
Grabbing:
Carbon-Steel Fiber, Titanium backed, $580
60%
25 + d10 DR
6 primary mags, 5 of combined secondary mags and grenades

**Comment by Wildlovelies** [comment]
lvl 5 stat point in fort was not added in

**Comment by Wildlovelies** [comment]
Ceramic Plate, Steel backed $220
        30%
        15 + d10 DR
4 primary  + 4 secondary mags

**Comment by Wildlovelies** [comment]
enough to fill all slots with extended mags at 10 credits per mag

**Comment by Wildlovelies** [comment]
lvl 8 would also take:
Acquired: Additional Plating
Prerequisites: Marksman, Gunslinger
Enables the use of Medium Armour by the listed classes.

Grabbing:
Carbon-Steel Fiber, Titanium backed,  $580
        60%
        25 + d10 DR
6 primary mags, 5 of combined secondary mags and grenades

**Comment by Wildlovelies** [comment]
this does not include gun kata

**Comment by Wildlovelies** [comment]
I followed Taylors new rule of 10 max stats at lvl 1, else I could have min maxed more to get 12 dex (4 bullets per half action) and add 4 shots per round more per gun additional 696 dmg, totaling 2784 damage per round

**Comment by 1sourcecontrol** [comment]
The updated weapon skill tree reduces the +30 feat damage bonus to +~9 damage. Totaling 66 damage compared to 87. Bringing it down to 1584 for dual-wielding during 4 firing actions. 
Throw in a multiplier for miss chance, 0.74 * 1584, it looks like ~1172 damage for 24 bullets. I think any reasonable machine-pistol dual-wield will yield over a thousand damage. If targets are sitting behind cover and have armor, you're looking at some significant reduction in effective damage.

You still have incredible damage output, but unless we nerf fully auto weapons and or dual wielding, I don't think that is going to change. The weapon revision I'm working on should definitely reduce the dps, tho: higher miss chances.

I'm not worried about this as much as enhanced/normal gun kata. The potential DPS with that is magazine limited, if I recall correctly.

**Comment by Wildlovelies** [comment]
enhanced gunkata, says you can reload
You are a master of pistols. Applies to an target within 35m. On a successful 

hit, you may continue rolling hits until you fail to hit. if you run out of 

rounds, you may reload mid-kata by passing a DC 75% DEX check. You may hit a 

target multiple times, but there must be one shot between shots fired at a 

single target. (Ask GM).

**Comment by bleehu** [comment]
<shrugs> I don't see the problem? He's really good at shooting lots of things. It's a good build, he's not too fragile and does a lot of damage at medium range. But he can't do his ultimate multi-target past 35m and cant blow things away past 55m. He's hard-countered by enemy snipers or controller mages, which should be par for the course by level 8 at least. On the flip side, he's the natural predator of swarming enemies and enemies that lurk or rush with melee attacks. Seems like a gunslinger doing what a gunslinger ought to do. And if you've taken the time to do all this math and tune your character really well, I think it's ok that he's a little stronger than average. But at the same time, he doesn't steal fun away from the soldier, medic or marksman on his team. I'm open to disagreement, but I suggest we close this one.

**Comment by 1sourcecontrol** [comment]
Do you think we could open another issue for enhanced gun kata level 2? I think it's too strong. Simply better firepower than anything else at that range when shooting 2+ targets

**Comment by bleehu** [comment]
@1sourcecontrol open that ticket please. 

I think we close this out. I think It's powerful, but not broken. If someone disagrees, we'll reopen it and fix it.
